app/components/WeatherDetails.js

```diff
@@ -0,0 +1,55 @@
+import { useSelector } from "react-redux";
+import WeatherSparkLine from "./WeatherSparkLine";
+import '../styles.css';
+
+
+// function that creates an object of conditions for each new city added
+const handleConditions = (location) => {
+  const locationConditions = {
+    id: location.city.id,
+    name: location.city.name,
```

Comment:
would recommend making variables as descriptive as possible - like cityName so you know exactly what kind of name this is